src/main/java/io/confluent/connect/jdbc/dialect/GenericDatabaseDialect.java

```diff
@@ -464,7 +464,7 @@ protected String[] tableTypes(
     Set<String> matchingTableTypes = new HashSet<>();
     try (ResultSet rs = metadata.getTableTypes()) {
       while (rs.next()) {
-        String tableType = rs.getString(1);
+        String tableType = rs.getString(1).trim();
```

Comment:
@ravib777 As mentioned [here](https://confluent.slack.com/archives/C045QDX4ECR/p1666777247811449?thread_ts=1665712622.404939&cid=C045QDX4ECR); let's keep this as a last resort option. But since you have taken a stab at it and raised the PR; I had a look at the same and have following comments:

1. If you see line 468 has a null check viz. "tableType != null" so now calling trim() before that null check can lead to a NPE. Please handle that null scenario.
2. I see that master branch has this line of code on [line#490](https://github.com/confluentinc/kafka-connect-jdbc/blob/master/src/main/java/io/confluent/connect/jdbc/dialect/GenericDatabaseDialect.java#L490); whereas I see your PR shows that as line 467! I am not sure why. Can you please double check?